core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/withEditor.ts

```diff
@@ -362,8 +371,15 @@ export function withEditor() {
                 setOgTags(ogTags: SeoMetaTags) {
                     patchState(store, { ogTags });
                 },
-                setPaletteOpen(paletteOpen: boolean) {
-                    patchState(store, { paletteOpen });
+                setPaletteTab(tab: UVE_PALETTE_TABS) {
+                    patchState(store, {
+                        palette: { open: true, currentTab: tab }
+                    });
+                },
+                setPaletteOpen(open: boolean) {
+                    patchState(store, {
+                        palette: { open, currentTab: store.palette().currentTab }
```

Comment:
spread the whole palette and override the open, that way if we add another prop we do not break anything